looprpc/client.proto

```diff
@@ -44,102 +34,63 @@ service SwapClient {
     ListSwaps returns a list of all currently known swaps and their current
     status.
     */
-    rpc ListSwaps (ListSwapsRequest) returns (ListSwapsResponse) {
-        option (google.api.http) = {
-            get: "/v1/loop/swaps"
-        };
-    }
+    rpc ListSwaps (ListSwapsRequest) returns (ListSwapsResponse);
 
     /* loop: `swapinfo`
     SwapInfo returns all known details about a single swap.
     */
-    rpc SwapInfo (SwapInfoRequest) returns (SwapStatus) {
-        option (google.api.http) = {
-            get: "/v1/loop/swap/{id}"
-        };
-    }
+    rpc SwapInfo (SwapInfoRequest) returns (SwapStatus);
 
     /* loop: `terms`
     LoopOutTerms returns the terms that the server enforces for a loop out swap.
     */
-    rpc LoopOutTerms (TermsRequest) returns (OutTermsResponse) {
-        option (google.api.http) = {
-            get: "/v1/loop/out/terms"
-        };
-    }
+    rpc LoopOutTerms (TermsRequest) returns (OutTermsResponse);
 
     /* loop: `quote`
     LoopOutQuote returns a quote for a loop out swap with the provided
     parameters.
     */
-    rpc LoopOutQuote (QuoteRequest) returns (OutQuoteResponse) {
-        option (google.api.http) = {
-            get: "/v1/loop/out/quote/{amt}"
-        };
-    }
+    rpc LoopOutQuote (QuoteRequest) returns (OutQuoteResponse);
 
     /* loop: `terms`
     GetTerms returns the terms that the server enforces for swaps.
     */
-    rpc GetLoopInTerms (TermsRequest) returns (InTermsResponse) {
-        option (google.api.http) = {
-            get: "/v1/loop/in/terms"
-        };
-    }
+    rpc GetLoopInTerms (TermsRequest) returns (InTermsResponse);
 
     /* loop: `quote`
     GetQuote returns a quote for a swap with the provided parameters.
     */
-    rpc GetLoopInQuote (QuoteRequest) returns (InQuoteResponse) {
-        option (google.api.http) = {
-            get: "/v1/loop/in/quote/{amt}"
-        };
-    }
+    rpc GetLoopInQuote (QuoteRequest) returns (InQuoteResponse);
 
     /* loop: `listauth`
     GetLsatTokens returns all LSAT tokens the daemon ever paid for.
     */
-    rpc GetLsatTokens (TokensRequest) returns (TokensResponse) {
-        option (google.api.http) = {
-            get: "/v1/lsat/tokens"
-        };
-    }
+    rpc GetLsatTokens (TokensRequest) returns (TokensResponse);
 
     /*
     GetLiquidityParams gets the parameters that the daemon's liquidity manager
     is currently configured with. This may be nil if nothing is configured.
     [EXPERIMENTAL]: endpoint is subject to change.
     */
-    rpc GetLiquidityParams (GetLiquidityParamsRequest) returns (LiquidityParameters) {
-        option (google.api.http) = {
-            get: "/v1/liquidity/params"
-        };
-    }
+    rpc GetLiquidityParams (GetLiquidityParamsRequest)
```

Comment:
just curious why don't we have something like `loop: `listauth`` in the docs? Because it's experimental?

Reply:
Good question. I added them now. Those tags are for the API docs only so it can also add the output of the given command to the generated docs. @carlaKC any objections to adding them?

Reply:
No objections!